tsl/src/continuous_aggs/create.c

```diff
@@ -557,9 +557,8 @@ mattablecolumninfo_create_materialization_table(MatTableColumnInfo *matcolinfo,
 	orig_ht = ts_hypertable_cache_get_entry(hcache, origquery_tblinfo->htoid, CACHE_FLAG_NONE);
 	if (hypertable_is_distributed(orig_ht))
 	{
-		remote_invalidation_log_add_entry(orig_ht,
+		remote_invalidation_log_add_entry(mat_ht,
```

Comment:
why is this mat_ht now?

Reply:
mat_ht should mean materialization hypertable

Reply:
Yes, we add a invalidation log entry for the original hypertable , not the materialization hypertable. So why is this changed to use mat_ht?